IFComp/lib/IFComp/Schema/Result/Prize.pm

```diff
@@ -147,6 +158,10 @@ __PACKAGE__->add_columns(
     },
     is_nullable => 0,
   },
+  "location",
+  { data_type => "char", is_nullable => 1, size => 64 },
+  "ships_internationally",
+  { data_type => "tinyint", is_nullable => 1 },
```

Comment:
I find nullable boolean fields can easily become a source of confusion. I'd suggest making this non-nullable with a default of 0.

Reply:
Good idea. In this case I'll set the default to 1, though, given that "I'm willing to ship internationally" is the current effective default that we want to provide an alternative to.